app.js

```diff
@@ -207,6 +208,7 @@ function countLiveNeighbors(currentGeneration, cellRow, cellColumn) {
 
       if (i === cellRow && j === cellColumn) continue;
       liveNeighbors += currentGeneration[neighborRow][neighborColumn];
+      // ^ the payoff to use 0/1 instead of true false
```

Comment:
As I was working through other sections, I was wondering why you wouldn't use true false bc then for clicking you can just negate with a !.

But this is a clever use for liveNeighbors